backend/azureblob/azureblob.go

```diff
@@ -217,6 +218,17 @@ This option controls how often unused buffers will be removed from the pool.`,
 				encoder.EncodeDel |
 				encoder.EncodeBackSlash |
 				encoder.EncodeRightPeriod),
+		}, {
+			Name: "public_access",
+			Help: `Public access level of a container: blob, container or private(default).
+
```

Comment:
Can you reformat this a bit please?

These docs should be hard wrapped at 80 characters so they view ok in terminals.

They are also markdown docs displayed in the config pages.